airbyte-webapp/src/components/ui/DatePicker/DatePicker.module.scss

```diff
@@ -0,0 +1,147 @@
+/* stylelint-disable selector-class-pattern, no-descending-specificity */
+
+@use "scss/colors";
+@use "scss/fonts";
+@use "scss/variables";
+@use "scss/z-indices";
+
+.wrapper {
+  position: relative;
+}
+
+.datepickerButtonContainer {
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 33px;
+  display: flex;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+  border: none;
+}
+
+.datepickerButton svg {
+  font-size: 14px;
+}
+
+.popup {
+  z-index: z-indices.$datepicker;
+}
+
+.input {
+  padding-right: 25px;
+}
+
+:global(.react-datepicker) {
+  display: flex;
+  color: colors.$dark-blue-900;
+  font-family: fonts.$primary;
+  border: variables.$border-thick solid colors.$grey-100;
+  border-radius: variables.$border-radius-md;
+  box-shadow: 2px 2px 20px -6px colors.$grey-200;
+}
+
+/** Main calendar area **/
+:global(.react-datepicker__header) {
```

Comment:
I saw that there are a few `xClassName` props on the ReactDatePicker component, which we generally prefer over using `:global`, correct? But, I'm guessing those don't give you enough granular control over the styling of each part of the datepicker, so that is why you needed to use `:global` here?